css/style.css

```diff
@@ -75,3 +111,24 @@ input {
   background-color: rgb(184, 14, 14);
   color: #fff;
 }
+
+.contact-header,
+.cont-info {
+  text-align: center;
+}
+
+.cont-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- Great job for implementing the navigation :+1: however you missed the wireframe at some point, and your submission form and the table list aren't followed the wireframe, please update to have a better design and fulfill the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md#project-requirements) :+1: 

#### given Wireframe
![Screenshot from 2023-02-15 23-47-18](https://user-images.githubusercontent.com/52010789/219155861-592b6d5e-8175-495d-8df0-c974bdd4e235.png)
#### your design
![Screenshot from 2023-02-15 23-46-33](https://user-images.githubusercontent.com/52010789/219155874-9cccbb60-b9de-4662-9fe6-31f15c658a89.png)
![Screenshot from 2023-02-15 23-46-20](https://user-images.githubusercontent.com/52010789/219155878-1344986b-2a9b-4d29-82a3-163f00d3c0f7.png)

Reply:
Hello @binhussen, After fixing that we had some problems with the linters, so if you can help us we will be so grateful